config/main.cpp

```diff
@@ -27,12 +27,23 @@
 
 #include <LXQt/SingleApplication>
 
+#include <QCommandLineParser>
 #include "main_window.h"
 
 int main(int argc, char *argv[])
 {
     LXQt::SingleApplication a(argc, argv);
 
+    QCommandLineParser parser;
+    parser.setApplicationDescription(QStringLiteral("LXQt Config Globalkeys "));
+    const QString VERINFO = LXQT_GLOBALKEYS_VERSION \
```

Comment:
Ok, when such changes make our life less miserable in the sense of future changes we must do this. Full Ack.